FeedAPIChallenge/Feed API/FeedImageItemMapper.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by André Bongartz on 09.09.21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal enum FeedImageItemMapper {
```

Comment:
`internal` is the default access level, so you don't need to write it explicitly.

It was written explicitly in the first lectures of the program for teaching purposes, but you don't need to do it in your code.

```suggestion
 enum FeedImageItemMapper {
```